turnitintooltwo_view.class.php

```diff
@@ -1281,39 +1303,46 @@ public function get_submission_inbox_row($cm, $turnitintooltwoassignment, $parts
                 // Output grademark icon.
                 $grade = '';
                 if (!is_null($submission->submission_grade) || $submission->submission_gmimaged != 0 || $istutor) {
-                    $grade = $OUTPUT->box(
-                        html_writer::tag('i', '', array('title' => get_string('submissiongrade', 'turnitintooltwo'),
-                            'class' => 'fa fa-pencil fa-lg gm-blue')), 'grademark_open ' . $class,
-                        'grademark_' . $submission->submission_objectid . '_' . $partid . '_' . $moodleuserid,
-                        array('title' => $CFG->wwwroot . '/mod/turnitintooltwo/view.php?id=' . $cm->id)
-                    );
+
+                $submissiongradeicon = html_writer::tag('i', '',
```

Comment:
A couple of points from an accessibility perspective:
The `<i>` tag is generally used to voice text in an alternate mode - e.g. "He named his car _'KillJoy'_ for obvious reasons." - surrounding the word "KillJoy" with `<i>` tags would allow a screenreader to render more realistic speech as intended by the author.
If one is merely after italicising something, then css `font-style` can be used.

Although, looking at the contents of this it appears that this `<i>` tag is being used as a placeholder for an icon. Might be worth testing with VoiceOver to see if the screenreader picks up the intonation of the tag when reading the title that you pass, because if it does then the table header row could sound a bit weird to blind users.

Reply:
Thanks, I'll take a look and test that.

Reply:
Yeah they were `<i>` tags because they are font awesome icons, they should be able to be changed to `<span>` without any problems depending on your voiceover testing.

Reply:
Tested this and whether using span or i tags the inbox doesn't tab along to the icon. The accessibility of the direct inbox will be looked into this quarter.